src/utils/cssRuleHasSelectorEndingWithColon.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Check whether a rule has a selector ending in a colon
+ *
+ * @param {Rule} postcss rule node
+ * @return {boolean} If `true`, the rule has a selector ending in a colon
+ */
+export default function (rule) {
+  const selector = rule.raws.selector && rule.raws.selector.raw || rule.selector
```

Comment:
Coolio. I'll make the change. I saw you making use of lodash's functions to clean things up e.g. using `includes`. I'm not very familiar with all of lodash's functions though and so if you spot any other opportunities please let me know.